addon.xml

```diff
@@ -10,7 +10,7 @@
     <extension point="xbmc.service" library="service.py"/>
 
     <extension point="xbmc.addon.metadata">
-        <summary lang="en_GB">Skip YouTube in-video sponsors</summary>
+        <summary lang="en_GB">Skip YouTube in-video sponsors (Port)</summary>
         <description lang="en_GB">
 SponsorBlock is a crowdsourced project to skip sponsor segments in YouTube videos.
```

Comment:
How about we add it to the description?

```suggestion
This is a port of the SponsorBlock browser extension.

SponsorBlock is a crowdsourced project to skip sponsor segments in YouTube videos.
```

Reply:
There's also the [`disclaimer`](https://kodi.wiki/view/Addon.xml#.3Cdisclaimer.3E) tag we could use:
```xml
<disclaimer lang="en_GB">This is an unofficial port</disclaimer>
```